game.rb

```diff
@@ -3,15 +3,30 @@
 
 class Game
 
-  def initialize io_handler, prompter
+  attr_reader :word, :letter
+
+  def initialize game_view, io_handler, prompter, validator
+    @game_view = game_view
     @io_handler = io_handler
     @prompter = prompter
+    @validator = validator
   end
 
   def game_handler
     @io_handler.printer(@prompter.game_greeting)
     @io_handler.printer(@prompter.prompt_for_word)
-    @io_handler.printer(@io_handler.word_blank(@io_handler.get_input))
+    while @validator.is_a_word?(@word=(@io_handler.get_input.upcase)) == false
+      @io_handler.printer(@prompter.prompt_for_word)
+    end
+    system "clear"
```

Comment:
I am curious about how you would be able to test this method without actually clearing the console... Were you having that problem?